src/identify/index.js

```diff
@@ -4,93 +4,111 @@
  */
 
 var Interactive = require('multistream-select').Interactive
-var EventEmmiter = require('events').EventEmitter
-var util = require('util')
 var protobufs = require('protocol-buffers-stream')
 var fs = require('fs')
 var schema = fs.readFileSync(__dirname + '/identify.proto')
 var v6 = require('ip-address').v6
-var Id = require('ipfs-peer-id')
+var Id = require('peer-id')
 var multiaddr = require('multiaddr')
 
-exports = module.exports = Identify
+exports = module.exports = identify
 
-util.inherits(Identify, EventEmmiter)
+var protoId = '/ipfs/identify/1.0.0'
 
-function Identify (swarm, peerSelf) {
-  var self = this
-  self.createProtoStream = protobufs(schema)
+exports.protoId = protoId
```

Comment:
check line 14 `exports = module.exports = identify` that is typically the pattern I go for :)